src/script/components/InputBar/InputBar.tsx

```diff
@@ -214,6 +217,23 @@ export const InputBar = ({
     ),
   });
 
+  const handleSaveEditorDraft = (replyId = '') => {
```

Comment:
Shouldn't it be like that? (looking at other usage of `replyId`):

```suggestion
  const handleSaveEditorDraft = (replyId?: string) => {
```

Reply:
We have to pass empty string for proper update of localStorage :( So I do it like that